src/labels/CustomLabels.labels

```diff
@@ -8432,6 +8432,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description for Create Constituent Records section in Get Started</shortDescription>
+        <value>People are the heart of any nonprofit. Explore Nonprofit Success Pack features to learn how to manage your constituents and their household, personal, and professional connections.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistCreateConstituentRecordsTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title for Create Constituent Records section in Gest Started</shortDescription>
+        <value>Create Constituent Records</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactOrganizationBlueButton</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Blue button text for Connect Contacts to Organization checklist item</shortDescription>
+        <value>Create Affiliations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactOrganizationDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description for Connect Contact to Organization checklist item in Get Started</shortDescription>
+        <value>Affiliations in NPSP let you capture important connections between your constituents and related organizations such as work, school, or business. Connect a Contact to an Organization using Organization Affiliations.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactOrganizationLink</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link text for Connect Contacts to Organization checklist item in Get Started</shortDescription>
+        <value>Learn more about Affiliations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactOrganizationTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title for Connect Contact to Organization checklist item in Get Started page</shortDescription>
+        <value>Connect a Contact to an Organization with Affiliations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactsRelationshipsBlueButton</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Blue button text for Connect Two Contacts with Relationships checklist item</shortDescription>
```

Comment:
Again, I would avoid the use of the color for accessibility reasons. Please use: Button text for Connect Two Contacts with Relationships checklist item